training/src/main/scala/com/airbnb/aerosolve/training/pipeline/ResultUtil.scala

```diff
@@ -0,0 +1,113 @@
+package com.airbnb.aerosolve.training.pipeline
+
+import com.airbnb.aerosolve.training._
+import scala.collection.mutable
+
+/*
+ * Writes to result file
+ *
+ * Format:
+    {
+    "HOLD_PRECISION_RECALL_AUC": 0.897908,
+    "HOLD_ACC": 0.897908,
+    ...
+    "HOLD_THRESHOLDS": [
+      0.021060,
+      0.313343,
+      0.897908,
+      1.190191,
+      1.482474
+    ],
+    "HOLD_RECALLS": [
+      0.021060,
+      0.313343,
+      0.897908,
+      1.190191,
+      1.482474
+    ],
+    "HOLD_PRECISIONS": [
+      0.021060,
+      0.313343,
+      0.897908,
+      1.190191,
+      1.482474
+    ],
+    "TRAIN_PRECISION_RECALL_AUC": 0.897908,
+    "TRAIN_ACC": 0.897908,
+    ...
+    "TRAIN_THRESHOLDS": [
+      0.021060,
+      0.313343,
+      0.897908,
+      1.190191,
+      1.482474
+    ],
+    "TRAIN_RECALLS": [
+      0.021060,
+      0.313343,
+      0.897908,
+      1.190191,
+      1.482474
+    ],
+    "TRAIN_PRECISIONS": [
+      0.021060,
+      0.313343,
+      0.897908,
+      1.190191,
+      1.482474
+    ]
+    }
+ *
+ */
+object ResultUtil {
+  def writeResults(resultsOutputPath: String,
```

Comment:
suggest you add unit test for ResultUtil in this PR, and make ResultUtil extendable on next PR